src/main/java/org/mapfish/print/config/ConfigFactory.java

```diff
@@ -2,11 +2,14 @@
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.IOException;
 import java.io.InputStream;
 
 import com.codahale.metrics.MetricRegistry;
-import org.ho.yaml.CustomYamlConfig;
-import org.ho.yaml.YamlConfig;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.MapperFeature;
```

Comment:
this comes from a transitive dependency, it would be good to add 
```xml
      <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-databind</artifactId>
        <version>${jackson-dataformat-yaml.version}</version>
      </dependency>
```
to the dependencies; also I would in that case rename the version variable `jackson-dataformat-yaml.version` to `jackson2.version`

Reply:
Done thanks.